packages/trader/src/Modules/Trading/Containers/purchase.tsx

```diff
@@ -124,7 +125,7 @@ const Purchase = observer(({ is_market_closed }) => {
             />
         );
     }
-    return components;
+    return <React.Fragment>{components}</React.Fragment>;
```

Comment:
@kate-deriv let's not wrap it into a React.Fragment and put: `const components: React.ReactNode[] = [];` at L44  instead. If it doesn't work let's assert an acceptable type. We had some issue in the past because of this when styles broke for components because React.Fragment tag was added just to satisfy TS.